src/lib/components/Access/ProtectionButtons.js

```diff
@@ -72,12 +72,10 @@ ProtectionButtonsComponent.defaultProps = {
 
 export class ProtectionButtons extends Component {
   render() {
-    const { fieldPath, active, disabled } = this.props;
+    const { fieldPath } = this.props;
 
-    const changed = !active || disabled ? { changed: true } : {};
     return (
-      <FastField
-        {...changed}
+      <Field
```

Comment:
This component was not re-rendering after the changes on the reducer.

It can be analysed later.